app/javascript/components/widgets/numbers/panel.jsx

```diff
@@ -1,6 +1,119 @@
 import React from 'react';
-import Wrapper from '../../panel-wrapper';
+import PropTypes from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+import { over, lensPath, inc } from 'ramda';
+import pluralize from 'pluralize';
+import {
+  colors,
+  weights,
+  fontSizes,
+  spacing,
+  fonts,
+  leftPanelWidth,
+} from '../../../lib/theme';
 
-export const Numbers = () => <Wrapper />;
+const LoadingMessage = () => <p>Loading...</p>;
+const ErrorMessage = ({ message }) => <p>Error: {message}</p>;
+ErrorMessage.propTypes = {
+  message: PropTypes.string.isRequired,
+};
+
+const getEventCounts = gql`
+  {
+    events {
+      count {
+        githubPull
+        githubCommit
+        slackMessage
+      }
+    }
+  }
+`;
+
+const NumbersTitle = styled.div`
+  color: ${colors.white};
+  font-size: ${fontSizes.medium};
+`;
+
+const BigNumber = styled.div`
+  color: ${colors.white};
+  weight: ${weights.bold};
+  font-size: ${fontSizes.huge};
+`;
+
+const BigNumberSubtitle = styled.div`
+  font-size: ${fontSizes.medium};
+`;
+
+const NumberWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  width: 5em;
+`;
+
+class SubscribedEvents extends React.PureComponent {
+  render() {
+    const { githubPull, githubCommit, slackMessage } = this.props;
+
+    const commitText = pluralize('Commit', githubCommit);
+    const requestText = pluralize('request', githubPull);
+    const messageText = pluralize('message', slackMessage);
+
+    return (
+      <div>
+        <NumbersTitle>This week at MojoTech</NumbersTitle>
+        <NumberWrapper>
+          <div>
+            <BigNumber>{githubCommit}</BigNumber>
+            <BigNumberSubtitle style={{ color: colors.pink }}>
+              {commitText}
+            </BigNumberSubtitle>
+          </div>
+          <div>
+            <BigNumber>{githubPull}</BigNumber>
+            <BigNumberSubtitle style={{ color: colors.yellow }}>
```

Comment:
The `chore: add colors for number panel to theme` commit (where these colors are added) comes _after_ this commit. I'd just reorder these two commits.

Alternatively: Some devs have strong feelings that a commit that introduces new code should also _use_ that new code, so some would advise that these two commits be squashed together.

I personally feel that keeping them separate helps me review and is arguably more atomic.

Reply:
Yeah, doing it before makes more sense. We talked through some commit styling things with Craig that I am working on. 👍